File: easy/python/issue10.py
#!/usr/bin/python2 
# Problem -> Create a function that takes a base 
#number and an exponent number and returns the calculation.

def power(base, exponent):
    x = 0
    y = 0
    while x < exponent:
        y += base * base
        x += 1
    return y 

print(power(2,3))

File: easy/c++/issue10.cpp
// Problem -> Reverse boolean value 

#include <iostream>

using namespace std; 

int reverse(bool value) {
	cout << value << endl; 
	return value; 
}
int main() {
	int value = reverse(false);
	if(value == 0) {
		cout << "True" << endl; 
	}else if(value == 1){
		cout << "False" << endl; 
	}
return 0; 
}